src/app/product/[productId]/page.tsx

```diff
@@ -0,0 +1,10 @@
+import { ProductDetail } from "@/components/ProductDetail/ProductDetail";
+
+export default function ProductPage({ params }: { params: { productId: string } }) {
+	const { productId } = params;
+	return (
+		<>
```

Comment:
When you only have one child, its not necessary to have the `<> </>` wrapper

Reply:
I'll have more components here like review section of the specific product